services/moneypenny/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "moneypenny.serviceAccountName" . }}
+  labels:
+    {{- include "moneypenny.labels" . | nindent 4 }}
+imagePullSecrets:
+  - name: "pull-secret"
```

Comment:
Similarly pointless here. This is already set on the deployments.

Reply:
imagePullSecrets removed